test/unit/reductions/grid_search/test_grid_search_regression.py

```diff
@@ -64,16 +64,6 @@ def test_bgl_unfair():
     best_predict = target.predict(test_X)
     assert np.allclose([-1.91764706,  9.61176471], best_predict)
 
-    all_predict = target.posterior_predict(test_X)
-    assert np.allclose([[3.2, 11.2],
-                        [-3.47346939, 10.64897959],
-                        [-2.68, 10.12],
-                        [-1.91764706, 9.61176471],
-                        [-1.18461538,  9.12307692],
-                        [-0.47924528,  8.65283019],
-                        [0.2, 0.7]],
-                       all_predict)
-
 
```

Comment:
it would be nice not to lose this test!

Reply:
But.... I thought I was supposed to get rid of `posterior_predict` for `GridSearch` ?

Reply:
Understood. Have reinstated the golden value test, but manually implementing `posterior_predict` for this test